index.bs

```diff
@@ -6219,6 +6200,48 @@ dictionary MLOperandDescriptor {
   </div>
 </details>
 
+Algorithms {#algorithms}
+=====================
+
+## Broadcasting ## {#algorithms-broadcasting}
+
+Broadcasting refers to how operations treat tensors with different shapes, and follow the precedent set by [[!numpy-broadcasting-rule]].
+
+<div algorithm>
+To <dfn data-lt="unidirectionally broadcasting the shapes">unidirectionally broadcast the shapes</dfn> |A| and |B|, perform the following steps. |A| and |B| are [=/lists=] of positive integers, representing the dimensions of tensors, and the steps return a new [=/list=] of positive integers, or failure.
+
+1. Let |sizeA| be the [=list/size=] of |A|.
+1. Let |sizeB| be the [=list/size=] of |B|.
+1. Let |output| be a new [=/list=].
```

Comment:
```suggestion
1. Let |outputShape| be a new [=/list=].
```
Just for added clarity, since `output` often is used for the *tensor*, and consistency with `outputShape` elsewhere in spec.